tf2onnx/tfonnx.py

```diff
@@ -292,6 +292,20 @@ def placeholder_op(ctx, node, name, args):
     return None
 
 
+def placeholder_with_default_op(ctx, node, name, args):
+    default_value_node = ctx.get_node_by_name(node.input[0])
+    if default_value_node.is_const():
+        # Clone the tensor, set its name as current node's output, add as initializer
+        t = onnx_pb.TensorProto()
```

Comment:
hm, the runtime supports this?

Reply:
do not really know yet...
there is bug for required input count check currently, and someone is fixing it.

the runtime should support it since it is totally legal according to onnx spec.

Reply:
but this would mean everything out there would run into this if it uses the placeholder_default

Reply:
i got your point. 
yes, it will crash on onnxruntime 0.1.3 when placeholder_default is used and user tried to feed input to override default value.

seems we should postpone on this pr for a while.

Reply:
I understand that's the runtime issue, right? the conversion did the right thing? we can postpone this until there is requirement for that. BTW, is there bug tracking the runtime issue?

Reply:
if you need it for some model we could enable or disable with
```ctx.is_target(TARGET_RS5) or `ctx.is_target(TARGET_RS6) ``` (should maybe make is_target accept a list).